rest/config_test.go

```diff
@@ -2135,7 +2135,8 @@ func TestWebhookFilterFunctionLoad(t *testing.T) {
 					Err:        test.errExpected,
 				}
 			}
-			ctx := &db.DatabaseContext{EventMgr: db.NewEventManager()}
+			terminator := make(chan bool)
```

Comment:
The event manager is never actually started in this test, but might as well throw in a defer close here to be defensive 

```suggestion
			terminator := make(chan bool)
			defer close(terminator)
```